Student/Widgets/GradesWidget/View/MediumLarge/GradeItemView.swift

```diff
@@ -0,0 +1,46 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2020-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import SwiftUI
+import WidgetKit
+
+struct GradeItemView: View {
+    var body: some View {
+        Link(destination: item.route) {
+            HStack {
+                Text("\(item.name)").lineLimit(2).font(.bold17).foregroundColor(item.color)
```

Comment:
This interpolation isn't helpful. It's going to lookup the localized string for `"%@"` then put your string in. `Text(item.name)` would skip creating, looking up, & formatting a `"%@"` entry in Localizable.strings.